nearcore/src/lib.rs

```diff
@@ -332,8 +333,18 @@ pub fn start_with_config(home_dir: &Path, config: NearConfig) -> NearNode {
     let view_client1 = view_client.clone().recipient();
     config.network_config.verify();
     let network_config = config.network_config;
+    let ibf_routing_pool = make_ibf_routing_pool();
+    #[cfg(all(feature = "json_rpc", feature = "adversarial"))]
+    let ibf_routing_pool2 = ibf_routing_pool.clone();
     let network_actor = PeerManagerActor::start_in_arbiter(&arbiter.handle(), move |_ctx| {
-        PeerManagerActor::new(store, network_config, client_actor1, view_client1).unwrap()
+        PeerManagerActor::new(
+            store,
+            network_config,
+            client_actor1,
+            view_client1,
+            ibf_routing_pool.clone(),
```

Comment:
I feel you don't need to clone it here and below since you already have two instances of `ibf_routing_pool`

Reply:
Thanks, fixed.